<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Australia</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The Australian Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://usegalaxy-au.github.io/index-ecology.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Australia" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-info" style="margin-top: 0.5em">
	<h4>New tools added</h4>
	
	
	<div><p>We have recently added some heatmapping and chimera detection tools to Galaxy Australia. See news item for details</p>
</div>
	
</div>





  <p>Welcome to <strong>Galaxy for Ecology</strong> – a web platform to get, process, analyze and visualize ecological data</p>

<p><img src="/assets/media//Galaxy-E-concarneau-team-2018-logo.gif" height="100px" alt="2018 Concarneau Galaxy-E team" /></p>

<ol id="markdown-toc">
  <li><a href="#getting-started" id="markdown-toc-getting-started">Getting started</a></li>
  <li><a href="#tools" id="markdown-toc-tools">Tools</a>    <ol>
      <li><a href="#tutorials" id="markdown-toc-tutorials">Tutorials</a></li>
    </ol>
  </li>
  <li><a href="#workflows" id="markdown-toc-workflows">Workflows</a>    <ol>
      <li><a href="#references" id="markdown-toc-references">References</a></li>
    </ol>
  </li>
</ol>

<h2 id="getting-started">Getting started</h2>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a href="https://ecology.usegalaxy.eu/tours/core.galaxy_ui" target="_top">a guided tour</a> through Galaxy’s user interface.</p>

<p>Want to learn about ecology analyses? Check our <a href="#tutorials">tutorials</a> or take one of our guided tour:</p>

<ul>
  <li>Introduction to species phenology study and abundance index as trends computation.</li>
</ul>

<h1 id="tools">Tools</h1>

<p>Almost 140 tools are proposed in this custom Galaxy instance. They were chosen for their use in exploitation of ecology data:</p>

<ul>
  <li><strong>General tools</strong>
    <ul>
      <li><strong>Text Manipulation</strong>: JQ process JSON, Replace Text, cast, melt, Subtract, Complement, Cluster, Replace text in a specific column, Replace parts of text, text reformatting, Text transformation, Unfold columns, Replace column, Add input name as column, Create text files, Sort a row, reverse a file, Compute an expression, Regex replace, Subtract Whole Dataset, Merge Columns, Unique occurences, Add column, Merge Columns together, Convert delimiters to TAB, Change Case, Trim characters, Secure Hash</li>
      <li><strong>Filter and Sort</strong>: Unique, Unique lines, Sort, Select random lines, Select first lines (head), Select last lines (tail), Remove beginning, Cut columns, Search in textfiles, XPath, Column arrange, Query tabular, Filter data on any column, Sort data, Select lines, Remove columns, Sort Column Order</li>
      <li><strong>Join, Substract and Group</strong>: Join two files, Multi-Join, Split file, Concatenate datasets, Paste two files side by side, Reverse, Transpose, Datamash, Subtract, Join two Datasets, Compare two Datasets, Group data by a column</li>
      <li><strong>File conversion</strong>: Tabular to CSV, CSV to Tabular, RData parser, RData reader, SQLite to tabular, Netcdf Read, Netcdf Metadata info, GDAL Translate</li>
      <li><strong>Graph / Display Data</strong>: Bar chart, Histogram, Histogram w ggplot2, Scatterplot, Scatterplot w ggplot2, Plotting tool for multiple series, Boxplot, heatmap2, Violin plot w ggplot2, Heatmap w ggplot, PCA w ggplot2, rtsne, Visualize hierarchical data with Krona, Venn Diagram</li>
    </ul>
  </li>
  <li><strong>Statistics tools</strong>
    <ul>
      <li><strong>Treat Samples</strong>: Generate random samples, Select max values Statistical</li>
      <li><strong>Pre-process</strong>: Line/Word/Character count, Count, Arithmetic Operations, Feature Selection, Preprocess raw feature vectors</li>
      <li><strong>Descriptive Stats</strong>: Summary Statistics, Hypothesis testing, Correlation</li>
      <li><strong>Classification</strong>: Numeric Clustering, Calculate metrics for classification, Nearest Neighbors Classification</li>
      <li><strong>Multi-dimensional Analysis</strong>: PCA, Kernel CCA, CCA, Generate a Matrix for PCA/LDA, Perform LDA, Draw ROC plot, Multivariate PCA PLS and OPLS</li>
      <li><strong>Classical Analysis</strong>: Univariate statistics, T Test for Two Samples, ANOVA, Wavelet variance, Evaluate pairwise distance, Sparse Matrix Functions</li>
      <li><strong>Model-based</strong>: Perform Best-subsets Regression, Hyperparameter Search, Pipeline Builder, Model validation, Generalized linear models, Discriminant analysis, MINE, Calculate metrics for regression performance, Ensemble methods</li>
    </ul>
  </li>
  <li><strong>Ecology tools</strong>
    <ul>
      <li><strong>Get Ecological data</strong>: Get species occurences data, Get protocoled data from Vigie-Nature ,Get climatic data from Worldclim</li>
      <li><strong>Graph / Display Ecological data</strong>: Compare sites</li>
      <li><strong>Animal Detection on Acoustic Recordings</strong>: Tadarida-d (from wav file to features), Tadarida-c (from features to species), Tadarida data cleaner, Tadarida identification integration, Advanced restitution: raw approach, Advanced restitution: summary</li>
      <li><strong>Phenology &amp; Trend computation</strong>: Flight curve, Abundance index, Expected temporal trend, Model temporal trend, Autocorrelation test, Linear regression adjusted, Plot abundance</li>
      <li><strong>GIS objects handling</strong>: GDAL Info, GDLA Translate, GDAL addo, GDAL Build VRT, GDAL Merge, GDAL Warp, OGR Info</li>
    </ul>
  </li>
</ul>

<h2 id="tutorials">Tutorials</h2>

<p>We are passionate about training. So we are working in close collaboration with the <a href="https://galaxyproject.org/teach/gtn/" target="_blank">Galaxy Training Network (GTN)</a> to develop training materials of data analyses based on Galaxy <a href="#batut2017community">(Batut <i>et al.</i>, 2017)</a>. These materials hosted on the GTN GitHub repository are available online at <a href="https://training.galaxyproject.org" target="_blank">https://training.galaxyproject.org</a>.</p>

<p>We then developed <a href="https://galaxyproject.github.io/training-material/topics/ecology/" target="_blank">several tutorials</a> and more will come:</p>

<ul>
  <li>Introduction to species phenology study and abundance index as trends computation</li>
  <li>Introduction to Species Distribution Modeling (SDM) using Wallace R Shiny app</li>
</ul>

<h1 id="workflows">Workflows</h1>

<ul>
  <li>Phenology study and abundance index as trends computation through regionalGAM</li>
</ul>

<h2 id="references">References</h2>
<ol class="bibliography"><li><span id="index-metagenomics-batut2017community">Batut,B. <i>et al.</i> (2017) Community-driven data analysis training for biology. <i>BioRxiv</i>, 225680.</span></li></ol>


<h2>Our Data Policy</h2>

<style>
    th, td {
        border-bottom: 1px solid #ddd;
        padding: 10px;
    }
    th {
        background-color: #f2f2f2;
    }

</style>

<p><strong>Galaxy Australia</strong> is designed for data analysis and not for
long term storage of data.</p>
<br />
<p>Use Galaxy Australia to host your input data only for the period required for
analysis. Also, remember to export/download your analysed data as this also
will not be stored beyond the limits set out below.</p>

<p>It is <u>your responsibility</u> as a user of the system to manage your own data
and routinely remove both your input and output data from this community system
to enable capacity for other users. Any data that is not removed by you within
a defined time period (see below) will be automatically and permanently deleted.
</p>

<p>Galaxy Australia maintains a collection of frequently used reference genomes
and annotation datasets. The inclusion of additional reference genomes and/or
annotation data on the system for community use can be <a href="https://docs.google.com/forms/d/e/1FAIpQLSdXuarvkzFA5kRqoCfO8uiUGAB0PplfR4yvAfpCPSpdMcehmA/viewform">
requested</a>. Galaxy Australia's hosting of all reference and annotation data
does not count to your quota and it is the best way to access
reference/annotation data.</p>

<h3>Data storage quotas and retention periods</h3><br />
<center>
<table>
    <tr>
        <th></th>
        <th>Storage quota</th>
        <th>Data retention period</th>
    </tr>
    <tr>
        <td><strong>Registered Australian researchers</strong></td><td>600GB</td><td>90 days</td>
    </tr>
    <tr>
        <td><strong>Other registered users</strong></td><td>100GB</td><td>90 days</td>
    </tr>
    <tr>
        <td><strong>Unregistered users</strong></td><td>5GB</td><td>NA</td>
    </tr>
</table>
</center>

<h3>Registered Users</h3>

<ul>
    <li><u>Registered Australian Researchers</u> are defined by registration
        Email from:
        <ul>
            <li>@domain.edu.au</li>
            <li>@domain.org.au</li>
            <li>@domain.edu - only in the case of known Australian Universities
                not on the .au domain</li>
        </ul>
    </li>
    <li><u>Other Registered Users</u> are defined as any registration Email
        from all other @domains</li>
    <li>Please contact us (help@genome.edu.au) if your institution does not
        conform to this rule but you understand it should be defined as
        performing publicly funded Australian research</li>
</ul>

<p>Registered users from Australian publicly funded research organisations have
a 600GB data storage quota. Other registered users have a 100GB quota. An
increased data storage quota can be <a href="https://docs.google.com/forms/d/e/1FAIpQLSeiw6ajmkezLCwbXc3OFQEU3Ai9hGnBd967u9YbQ8ANPgvatA/viewform">
requested</a> for a limited time period in special cases.</p>


<p>Registered User's data (i.e. datasets, histories) will be available on the
system for 90 days from the point of upload or creation. Within this period,
any data marked by a Registered User as "deleted" will be permanently removed
within 7 days. If a registered user "purges" the dataset, it will be removed
immediately and permanently.</p>

<h3>Unregistered Users</h3>

<p>Processed data will only be accessible during one browser session, using a
browser cookie to identify an Unregistered User's data. This cookie is not used
for any other purposes (e.g. tracking or analytics.)</p>


<h3>What does it mean when I go over quota?</h3>

<p>You data and histories are still accessible, but you not be able to run new
jobs or import more data. If you know in advance then take advantage of
<a href="https://docs.google.com/forms/d/e/1FAIpQLSeiw6ajmkezLCwbXc3OFQEU3Ai9hGnBd967u9YbQ8ANPgvatA/viewform">
requesting</a> more analysis storage.</p>

<div>
    <center>
        <iframe style="border: 0px" width="90%" height="200px" src="https://stats.genome.edu.au/d-solo/-D4mtTAik/for_embedding?refresh=10s&orgId=1&panelId=2" ></iframe>
    </center>
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    





<footer class="navbar-default">
    <div class="container">
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.org.au is maintained largely by the <a href="/people">Australian Galaxy Team</a> including staff from QCIF, UQ-RCC and Melbourne Bioinformatics.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>, unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.org.au">contact@usegalaxy.org.au</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-au">usegalaxy-au</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyaustralia">galaxyaustralia</a></li>
                    
                      <!-- <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li> -->
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
