BEGIN:VCALENDAR
PRODID:-//jekyll//NONSGML v1.0//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:HPC COVID-19 Australian Response Showcase
X-WR-TIMEZONE:Europe/Berlin
BEGIN:VEVENT
ORGANIZER;CN=Pawsey Supercomputing Centre:MAILTO:admin@pawsey.org.au?subject=HPC%20COVID-19%20Australian%20Response%20Showcase
DTSTART;VALUE=DATE:20200731
DTEND;VALUE=DATE:20200731
DTSTAMP;VALUE=DATE:20200731T000000Z
CREATED;VALUE=DATE:20200709T000000Z
UID:2020073120200731@usegalaxy.eu
DESCRIPTION:Friday July 31 15:00-17:00 (AEST) 13:00-15:00 (AWST)\n\n<h4 id="principal-investigator-dr-gareth-price-qcif-facility-for-advanced-bioinformatics">PRINCIPAL INVESTIGATOR: Dr Gareth Price, QCIF Facility for Advanced Bioinformatics.</h4>Gareth manages the diverse spectrum of researcher lead questions involving genomic data, provides training in genomic data analysis, as well as leading the Galaxy Australia as Platform Manager. Gareth has 20 years’ experience as a Bioinformatician and Genomics Scientist.\n\nPROJECT: Galaxy Australia COVID-19 Dedicated Pulsar\n\nGalaxy Australia relies on remote (to head node) deployments called Pulsar to increase the range and number of jobs that can be run on the service. The team has been allocated resources on the Nimbus cloud to deploy a dedicated COVID-19 Pulsar as part of Galaxy Australia at the Pawsey Centre that allows Galaxy users to rapidly analyse their data on published tools/workflows to further research into SARS-CoV-2.\n\n<h4 id="principal-investigator-aprof-megan-omara-australian-national-university">PRINCIPAL INVESTIGATOR: A/Prof. Megan O’Mara, Australian National University</h4>Megan studied medical science and worked in pathology labs before returning to university to study physics. Her research uses computer simulations to understand how proteins in the cell membrane act as “molecular machines” to transmit information across the membrane; and how we can design more effective pharmaceutical drugs to treat conditions such as chronic pain, chemotherapy resistance, antibiotic resistance and more recently, Covid-19.\n\nPROJECT: Using large-scale molecular dynamics for rational drug designThis research uses simulations of the around 800,000 atoms that make up a key receptor of the human body to understand exactly how the coronavirus uses it to invade human cells. It is only with high-resolution modelling that accurately replicates the true behaviours of these receptors that we can figure out where vulnerabilities in the virus’ binding process are. Targeting the interaction between the human receptors and the coronavirus binding protein might well be a useful direction for drug design. This project will produce world-first vital information about regions of the receptors that could be potential vaccine or drug targets. Using 48 processors running for 19 days for each of 64 molecular simulations, this research will spend around 13 million units of computing time in the coming months. This amount of high-performance computing is only available on NCI’s new Gadi supercomputer.\n\n<h4 id="principal-investigator-dr-tom-karagiannis-monash-university">PRINCIPAL INVESTIGATOR: Dr Tom Karagiannis Monash University</h4>Tom’s PhD studies were aimed at developing DNA-targeted cancer therapies, these were continued during post-doctoral studies in the Molecular Radiation Biology Laboratory.  Tom headed the Epigenomic Medicine Laboratory at the Alfred Medical Research and Education Precinct; Baker Heart and Diabetes Institute and currently at Monash University. In addition, he has been involved in teaching at the Department of Pathology, University of Melbourne.\n\nPROJECT: Molecular modelling COVID-19 targets\n\nThe team has been allocated GPU resources on Topaz for COVID-19 targets molecular modelling. In the absence of a vaccine, one of the key immediate research areas is repurposing existing compounds with potential antiviral effects. The research project aims to provide a molecular basis for known antivirals and identify any new ligands which may offer a protective effect.\n\n<h4 id="principal-investigator-associate-professor-michael-wise-the-university-of-western-australia">PRINCIPAL INVESTIGATOR: Associate Professor Michael Wise, The University of Western Australia</h4>Michael is a computer scientist working in computational biology since the early 1990s. His early contributions related to in silico investigations ranging from the plant desiccation tolerance protein LEA to low complexity and natively unfolded proteins. For the last several years he is mostly involved with microbial informatics, and is particularly interested in the evolutionary biology of microbial species, and viruses\n\nPROJECT: SARS, SARS-Cov-2 and MERS are the Same Viral Species, Clades within Bat Beta-Coronaviruses\n\nSARS-CoV-2 and SARS are human coronaviruses of zoonotic origin first transferred from bats to humans in China, regarded as sister clades within the viral species Severe Acute Respiratory Syndrome-Related Coronavirus. A chain of bat beta-coronavirus strains links SARS and SARS-CoV-2. Here I report phylogenetic tree reconstructions in which SARS and SARS-CoV-2 are placed together with related bat strains. We see that single-species models are favoured over speciation models. This is also the case when the phylogenetic trees  are computed for MERS and its related beta-coronaviruses in camels, bats and hedgehogs.\n\nRemarkably, single species models still apply even when the datasets are combined across their shared core-proteome. Dated phylogenetic reconstructions suggest that both SARS-like and MERS-like beta-coronaviruses have been circulating for many years, and the population has been largely constant.\n\n<h2 id="more-information-and-registration-details"><a href="https://pawsey.org.au/event/pawsey-friday-covid19/" target="_blank">More information and registration details</a></h2>
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:HPC COVID-19 Australian Response Showcase
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
